.travis.yml

```diff
@@ -3,30 +3,12 @@ rust:
   - stable
   - beta
   - nightly
-  - 1.24.0
+  - 1.29.0
 env:
   - BITCOINVERSION=0.18.0
   - BITCOINVERSION=0.18.1
   - BITCOINVERSION=0.19.0.1
   - BITCOINVERSION=0.19.1
```

Comment:
we already know `BITCOINVERSION=0.20.1` it's not compatible or we should add it?

Reply:
it may be out of scope...

Reply:
There's another MR for that. That's kinda why I did this one right now.

Reply:
sorry, I missed that